packages/rum-core/src/tools/syntheticsContext.ts

```diff
@@ -0,0 +1,30 @@
+import { getCookie } from '@datadog/browser-core'
```

Comment:
`tools/syntheticsContext.ts`, should it be consider as a tool?

Reply:
Would you rather put it in `domain`? I thought it wasn't directly related to how RUM works, but happy to move it elsewhere.

Reply:
Why not to have tools, but I would also think of moving other components like `ContextHistory` which also looks like tools.

Reply:
IMO
- syntheticsContext -> domain
- contextHistory -> tools